crm_facebook_leads/models/lead.py

```diff
@@ -253,7 +253,8 @@ def lead_processing(self, response, form):
             with self.env.cr.savepoint():
                 for lead in data:
                     lead = self.process_lead_field_data(lead)
-                    if not self.search([('facebook_lead_id', '=', lead.get('id'))]):
+                    if not self.search([('facebook_lead_id', '=', lead.get('id')),
```

Comment:
use with_context(active_test=false) in order to include archived records

see next example
https://github.com/odoo/odoo/blob/338f8d843832c339c52c00c7f4c41b5ab2d696ed/addons/project/wizard/project_delete_wizard.py#L16

Also could you please add limit=1

Reply:
Thanks so much for the feedback, of course, allow me some minutes to fix it.

Reply:
Done, please check